addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   - XPath Injection
     - Added evidence
 - The Source Code Disclosure - File Inclusion scan rule was modified to make use of the Dice algorithm for calculating the match percentage, thus improving its performance.
-- Maintenance changes.
+- Maintenance changes (some changes impact the visibility of variables/constants, which may impact third party add-ons or scripts).
```

Comment:
This is very unlikely (IMO could be omitted), the scan rules are not expected to be used directly.